packages/allure-jest/README.md

```diff
@@ -2,21 +2,29 @@
 
 Allure integration for jest framework
 
+> **Warning**
+> This integration don't work with default runner since jest 17+.
+> To use this integration after jest 17 you need to change default runner to `Jasmine`.
+
 # How to use
 
+You can find example setup and usage in this [repo](https://github.com/vovsemenv/allure-jest-example)
+
 ## Default test runner (Jasmine)
 
 By default jest uses jasmine as test runner, so you can configure allure-jasmine reporter for it:
```

Comment:
now circus is used by default, so we need to add example how to change the runner, and what to do if you want to use circus

Reply:
fixed